pages/api/Schema/scalarResolvers.ts

```diff
@@ -0,0 +1,19 @@
+import { GraphQLScalarType } from 'graphql';
+import { Resolvers } from '../graphql-rest.page.generated';
+
+const ISO8601DateScalar = new GraphQLScalarType({
+  name: 'ISO8601Date',
+  description: 'An ISO 8601-encoded date',
+});
+
+const ISO8601DateTimeScalar = new GraphQLScalarType({
+  name: 'ISO8601DateTime',
+  description: 'An ISO 8601-encoded date time',
+});
+
+export const ScalarResolvers: Resolvers = {
```

Comment:
I was getting some errors with the custom scalars and added some code that seems to have made them work. Not quite sure if there's a simpler solution.

Reply:
So I think what we need to do is make a ```Schema/commonTypes``` folder and extract these custom scalars into that. Which is essentially what you did here lol. The error you got could have been because it was getting declared in more than one spot? https://github.com/CruGlobal/mpdx-react/blob/50024c25cfe7ac741daadfcf22b90a3c4e4796de/pages/api/Schema/TaskAnalytics/taskAnalytics.graphql#L16 Robert ran into a similar issue in his coaching questions pr, if I'm remembering correctly.